proposals/stage-1/TRAVEL_FUND/MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,169 @@
+significant impact than attending. However, individuals applying for the travel fund to speak or participate in panels should confirm that the event organizers cannot cover the travel expenses before submitting a request.
+
+#### Outreach
+* Preference is given to underserved communities. More specifically, within a subject the foundation is trying to promote awareness
+about (e.g. general knowledge about the Foundation) there are people, communities, and geographies where that knowledge is not very
+widespread. This is where preference will be given. For instance, an event in San Francisco would be less attractive than an event of the same size in Kansas City.
+
+#### Cost
+* The larger the stipend the more critically the travel fund admins will consider the request and application.
+The budget for this program is a finite resource.
+
+#### Equity
+* Preference is given to individuals who cannot receive travel funding from their employers or individuals who have not previously received a stipend.
+
+## Reimbursement
+
+So that we can reimburse you more quickly, we strongly encourage you to use
+[Expensify](https://expensify.com) to organize and submit your receipts.  Please
+see our [guide to creating and configuring a free Expensify
+account](./reimbursement_process.pdf).
+
+Here are the highlights of the process:
+
+* A free [Expensify](https://www.expensify.com/) account is sufficient.
+* You can use the Android and iOS apps to capture receipts while you are
+  traveling.  If you do not use the app, you will need to upload photos of your
+  receipts after you return.
+* When you add an expense, please use local currency so that it matches the
+  amount on the receipt.
+* Once you have uploaded your receipts, please add them to a report and submit
+  it with [operations@openjsf.org](mailto:operations@openjsf.org) on cc, and the
+  Memo field set to "YOURNAME - TRIPNAME".
+
+When we receive your report, we will send you a form via DocuSign to collect
+your payment details.  At that point, you're done.
+
+### Alternate process
+
+If you decide not to use Expensify, the reimbursement process is manual and may
+take longer for us to process.  You can email a zip file of your receipts and
+the completed [Expense Report](./expense-report-template.xls?raw=true), renamed
+as ExpenseReport-OpenJS-EVENTNAME-YOURNAME-YYYYMM.xls, to
+[operations@openjsf.org](mailto:operations@openjsf.org).  Please send the file
+as a spreadsheet, not as a PDF.  We will send it back to you for signature in
+DocuSign.
+
+### Important:
+
+* Expense reports must be submitted within 30 days of event date. Any reports submitted after 30 days may not be reimbursed.
+* Disbursements are generally processed within 30 days.
+* We are unable to pre-pay for expenses.
+* The Cross Project Council representative to the Board of Directors will report back amounts consumed from the travel allocation on a monthly basis to the Board.
+
+## Trip report
```

Comment:
If we remove the trip report request above we should remove this suggestions

Alternatively we can deep link

Reply:
Removed.